Library/Homebrew/software_spec.rb

```diff
@@ -344,3 +344,17 @@ def checksums
     checksums
   end
 end
+
+class PourBottleCheck
+  def initialize(formula)
+    @formula = formula
+  end
+
+  def reason(reason)
+    @formula.pour_bottle_check_unsatisfied_reason(reason)
```

Comment:
Oh, right; I wasn't even aware that `attr_rw` is a custom thing. Thanks for the clarification!